custom_components/ocpp/enums.py

```diff
@@ -34,6 +34,10 @@ class HAChargerStatuses(str, Enum):
     firmware_status = "Status.Firmware"
     reconnects = "Reconnects"
     id_tag = "Id.Tag"
+    current_import = "Current.Import"
```

Comment:
These changes are not needed, please remove from the PR. See the upstream enums  repository https://github.com/mobilityhouse/ocpp